x-pack/plugins/cases/common/constants.ts

```diff
@@ -103,14 +103,17 @@ export const GENERAL_CASES_OWNER = APP_ID;
 
 export const OWNER_INFO = {
   [SECURITY_SOLUTION_OWNER]: {
+    appId: 'securitySolutionUI',
```

Comment:
Can we import these values from their plugins? Or would that cause circular dependencies?

Reply:
I did not want to increase the bundle size which is why I did not import them.